### ISSUE TITLE
an orb of black dragonkind

### ISSUE THREAD

**Issue by jokkmeister** [torilmud/issues#508]
This item needs a serious upgrade to its power. Now it is a called single target blind per 12 minutes with a duration of only one minute and which is also saved very easily even by lower level mobs. This power is real shitty also compared with the other orbs we've seen so far with enchanter one being awesome as blend cloak and invoker one affecting two creatures with a much better effect.

My suggestion is to make it one of the following two:

Single target blind with a cooldown of 12 mins as is today but with the same effect as blind proc of Bane stiletto, ie better duration and no save.

or

Make it same as it is now only affect all the aggressive mobs in the room.

**Comment by Casorus** [comment]
Agreed, as it is now it may as well not even be there.  It just doesn't land, and a one minute blind pales in comparison to the powers of the other orbs.  An item that blinds is great for liches, and I'd like it to stay as a blind but it's in dire need of an upgrade.
